### PR TITLE
Feature: Rush

### PR DESCRIPTION
**Pull request by MartynasZilinskas**

### CODE REVIEW

File: packages/ts-docs-gen/package.json [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    "pretest": "npm run build -- --noEmit && npm run build-tests",
+    "test": "npm run tslint && jest && echo All tests passed successfully.",
+    "test-watch": "start npm run build-tests -- --watchAll && jest --watchAll",
+    "jest": "jest --maxWorkers=4",
+    "tslint": "tslint --project . --config ./tslint.json && echo Successfully passed tslint test.",
+    "build": "tsc -p .",
+    "build-tests": "cd tests && ts-node ./build-tests.ts -p ./"
+  },
+  "engine": "node >= 7.5.0",
+  "author": "",
```
Comment (DeividasBakanas) [inline packages/ts-docs-gen/package.json]: Author missing: 
```
simplrjs <simplr@quatrodev.com> (https://github.com/simplrjs)
```

File: packages/ts-docs-gen/src/generator.ts [hunk 2]
```diff
@@ -72,7 +72,7 @@ export class Generator {
     public PrintToFiles(): void {
         // =====================================
         //
-        // First step: Preparing to what files we want to write. 
+        // First step: Preparing to what files we want to write.
```
Comment (DeividasBakanas) [inline packages/ts-docs-gen/src/generator.ts]: Maybe:
```
Preparing files we want to write / output / fill.
```

**Review by DeividasBakanas — CHANGES_REQUESTED** [review]

**Review by DonatasBrazionis — APPROVED** [review]

File: packages/ts-docs-gen/package.json (hunk 1 above)
Comment (MartynasZilinskas) [reply packages/ts-docs-gen/package.json]: Added.

File: packages/ts-docs-gen/src/generator.ts (hunk 2 above)
Comment (MartynasZilinskas) [reply packages/ts-docs-gen/src/generator.ts]: Fixed.

**Review by DeividasBakanas — APPROVED** [review]

**Pull request merged**